src/libsyntax/diagnostic.rs

```diff
@@ -293,8 +293,8 @@ impl fmt::Display for Level {
 
         match *self {
             Bug => "error: internal compiler error".fmt(f),
-            Fatal | Error => "error".fmt(f),
-            Warning => "warning".fmt(f),
+            Fatal | Error => "\n  error".fmt(f),
+            Warning => "\n  warning".fmt(f),
```

Comment:
I really like the relative paths because printing the full path leaks paths that don't exist, ie, to the buildbot that the stdlib was built on. This is annoying both because it is a nonexistant path, but because it includes "slave", which is really awkward.